api/specs/webserver/v0/openapi-share.yaml

```diff
@@ -0,0 +1,62 @@
+openapi: 3.0.0
+info:
+  title: resource sharing API
+  version: 0.1.0
+  description: 'Operations on sharing studies'
+servers:
+  - description: API server
+    url: '/v0'
+paths:
+  /share/study/{study_id}:
+    get:
+      summary: Gets sharing tokens for the given study_id
+      operationId: get_share_study_tokens
+      parameters:
+        - $ref: './openapi-share.yaml#/components/parameters/StudyId'
+      responses:
+        '200':
+          description: got tokens for sharing study
+          content:
+            application/json:
+              schema:
+                $ref: './components/schemas/share_study_tokens.yaml#/ShareStudyTokensEnveloped'
+        default:
+          $ref: './openapi.yaml#/components/responses/DefaultErrorResponse'
```

Comment:
why not use the /projects endpoint as well?
```
/projects/{project_id}/share_token:
  get:
```

Reply:
I tend to agree with SAN. but would call it ``/project/{project_id}/token?type=export`` instead. This way resource path remains with nouns and actions/flavors are passed via query

Reply:
Because the idea was to extend the share capability to not only studies but also services.

Reply:
Moved to /project/{project_id}/token?type=export